server/src/main/java/org/opensearch/common/settings/Setting.java

```diff
@@ -2073,6 +2073,23 @@ public static Setting<TimeValue> timeSetting(
         );
     }
 
+    public static Setting<TimeValue> timeSetting(
```

Comment:
Yes, this is used.

Reply:
It's used by the other newly introduced timeSetting() overload, but does anything actually use the variant the passes a Settings->TimeValue function parameter? Not a big deal, but generally good to avoid adding unused APIs.

Reply:
Got it. Incorporated your comments.